configure.ac

```diff
@@ -423,15 +423,9 @@ AC_SUBST(LIBCTL_CPPFLAGS)
 ##############################################################################
 
 # check for -lctlgeom:
-AC_CHECK_LIB(ctlgeom, vector3_dot, [have_libctlgeom=yes], [have_libctlgeom=no])
+AC_CHECK_LIB(ctlgeom, vector3_dot, [have_libctlgeom=yes; LIBS="-lctlgeom $LIBS"], [have_libctlgeom=no])
```

Comment:
This doesn't seem right to me, since we don't want to link `-lctlgeom` from the `meep` executable that uses Scheme.   That's why I had the separate `LIBCTLGEOM_LIBS` variable.

Reply:
Oh, nevermind, this is correct.   Now that we have included libmeepgeom in libmeep, we need to always link libctlgeom.

I think there is no conflict with Scheme because, although Scheme compiles its own version of the libctlgeom routines, it does so in C++ within a `ctlio::` namespace?  I'm not sure how it gets the right version of functions like `object_of_point_in_tree`, that aren't namespaced in `geom.c`, however … maybe the linker does the right thing because `geom.o` is listed before `-lctlgeom` in the link command for `meep`?